app/views/communities/mod.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="row">
+    <div class="col-3">
+      <div class="text-dark mb-3">
+        <div class="list-group" id="list-tab" role="tablist">
+          <li class="list-group-item text-white bg-secondary">Mod Operations</li>
+            <li class="list-group-item border-left border-danger"><%= link_to "Ban Users", mod_path(mod: "Banusers") %></li>
+            <li class="list-group-item"><%= link_to "Flairs" , mod_path(mod: "flairs") %></li>
+        </div>
+      </div>
+    </div>
+  <% if params[:mod] == "Banusers" %>
+    <%= render partial: "banusers" %> 
+  <%elsif params[:mod] == "flairs" %>
+    <%= render partial: "flairs" %>
+  <% end %>
```

Comment:
Can you refactor this code ?

Reply:
Both the conditions are necessary as iam using buttons  to trigger popup and both are different popups